lms/templates/verify_student/incourse_reverify.underscore

```diff
@@ -0,0 +1,31 @@
+<div id="wrapper-facephoto" class="wrapper-view block-photo face-photo-step">
+  <div class="facephoto view">
+    <h3 class="title"><%- gettext( "Take Your Photo" ) %></h3>
+    <div class="instruction">
+      <p><%= gettext("Course key: ") + courseKey %></p>
+      <p><%= gettext("Checkpoint name: ") + checkpointName %></p>
```

Comment:
@wedaly Is there a reason the text for re-verification should differ from the text for original verification? I'm all for changing this text, believe me--I actually wrote some replacement text for this several months ago but those changes didn't make it into the platform--but I don't want to create a mismatch if the text for both processes should be the same. (If I can submit some new text for re-verification and it gets used for verification as well, I'm A-OK with that.)